augur/tasks/github/facade_github/tasks.py

```diff
@@ -177,15 +177,19 @@ def process_commit_metadata(contributorQueue,repo_id):
             
             #Executes an upsert with sqlalchemy 
             cntrb_natural_keys = ['cntrb_login']
```

Comment:
This cntrb natural keys needs to be `cntrb_id`

Reply:
@IsaacMilarky : What @ABrain7710 says here makes sense to me, and I am also wondering if there is a reason we need this to be the natural key. When we first get commit users we don't have the cntrb_login, so I think the cntrb_id is the "unique thing" ... let me know.